datasets/flwr_datasets/merge_resplitter_test.py

```diff
@@ -0,0 +1,129 @@
+"""Resplitter tests."""
+import unittest
+from typing import Dict, Tuple
```

Comment:
Copyright notices is missing

```suggestion
"""Resplitter tests."""


import unittest
from typing import Dict, Tuple
```